src/ui/screens/home/ViewServers.qml

```diff
@@ -131,6 +150,27 @@ Item {
         }
     }
 
+    Loader {
```

Comment:
A 'HelpSheetLoader' component is starting to sound better and better.

Reply:
yea... or even just a loader as the base for all sheets. will do this for all in a fast follow